src/testUtils/awala/parcel.ts

```diff
@@ -29,31 +60,48 @@ async function generateStubNodeCertificate(
   });
 }
 
-async function generateParcelPayload(recipientIdCertificate: Certificate): Promise<Buffer> {
+async function generateSessionlessParcelPayload(
+  recipientIdCertificate: Certificate,
+): Promise<Buffer> {
   const serviceMessageEncrypted = await SessionlessEnvelopedData.encrypt(
```

Comment:
I missed this in a previews review.

There such be no use of of the `SessionlessEnvelopedData` anywhere in this repo. As you may remember from an exception you shared with me, that class is in the process of being removed across the board.

We should be using `SessionEnvelopedData` exclusively.